testing/darktable/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Kevin Daudt <ops@ikke.info>
+# Maintainer: Kevin Daudt <ops@ikke.info>
+_commit=de8ac28bbb45db1b3eb9b0d51d5ec1288cdfc9dc
+pkgname=darktable
+pkgver=2.1.0.1928
+pkgrel=1
+pkgdesc="an open source photography workflow application and raw developer"
+url="http://www.darktable.org/"
+arch="all"
+license="GPL3"
+depends="libxslt gtk+3.0-dev libgphoto2-dev libwebp-dev
```

Comment:
This doesn’t look right. Why darktable depends on -dev packages (i.e. header files and similar) in runtime?